share/get.go

```diff
@@ -66,14 +66,14 @@ func GetSharesByNamespace(
 	shares := make([]Share, len(leaves))
 	for i, leaf := range leaves {
 		if leaf != nil {
-			shares[i] = leafToShare(leaf)
+			shares[i] = LeafToShare(leaf)
 		}
 	}
 	return shares, data.Proof(), err
 }
 
-// leafToShare converts an NMT leaf into a Share.
-func leafToShare(nd format.Node) Share {
+// LeafToShare converts an NMT leaf into a Share.
+func LeafToShare(nd format.Node) Share {
```

Comment:
why?

Reply:
https://github.com/celestiaorg/celestia-node/blob/bbe68bba7d3fe233b1b3e0afa08ac54681d13b1f/share/get.go#L69

Reply:
> why?

Did not want to copy-paste the same method

Reply:
it's an artifact from the previous version of the blob service implementation.